source/sass/buttons.scss

```diff
@@ -62,81 +77,86 @@ button,
 .btn-primary {
   border: none;
 
-  $default: (
-    "color": $white,
-    "background-color": $blue,
-    "border-color": none
-  );
-
-  $default-hover: (
-    "color": $white,
-    "background-color": $dark-blue,
-    "border-color": none
+  $regular-mode: (
+    "default": (
+      "color": $white,
+      "background-color": $blue,
+      "border-color": transparent
```

Comment:
Do we really want `border-color: transparent`, rather than `border: none`? (do we still rely on the border's width to affect positioning?)

Reply:
`none` is not a valid value for `border-color`. We can also update `border-color` to the same color variable `background-color` is set to, but I find setting it to`transparent` is more easy to read (and parse in my head).

Reply:
I removed the `border-color` rules for `.btn-primary` as it by default is border-less (line 78). 

For `btn-secondary`:
`none` is not a valid value for `border-color`. We can also update `border-color` to the same colour variable `background-color` is set to, but I find setting it to `transparent` is easier to read (and parse in my head).

Reply:
okay, so we do intentionally want to keep the border there, just acting as another way to force a margin?

Reply:
Yes, some button states have border so I thought keeping them consistent is easier to manage in code.

<img width="705" alt="Screen Shot 2019-09-04 at 12 45 44 PM" src="https://user-images.githubusercontent.com/2492510/64286420-47a70580-cf12-11e9-8c1d-bd5ea75b06b6.png">

Reply:
Yes, default state for btn-secondary has border. If we remove border for the hover and active state, button position will shift as its state change from default to hover/active.

<img width="705" alt="Screen Shot 2019-09-04 at 12 45 44 PM" src="https://user-images.githubusercontent.com/2492510/64286420-47a70580-cf12-11e9-8c1d-bd5ea75b06b6.png">